config.xml

```diff
@@ -80,17 +80,39 @@
     <icon src="resources/android/icon/drawable-xhdpi-icon.png" />
     <allow-navigation href="http://10.0.0.175:8100" />
     <allow-navigation href="http://10.0.0.220:8100" />
-    <plugin name="cordova-plugin-console" spec="^1.0.5" />
-    <plugin name="cordova-plugin-device" spec="^1.1.4" />
-    <plugin name="cordova-plugin-geolocation" spec="^2.4.3" />
     <gap:plugin name="cordova-plugin-googlemaps" source="git" spec="https://github.com/mapsplugin/cordova-plugin-googlemaps">
         <param name="API_KEY_FOR_ANDROID" value="AIzaSyDa1_q4_YaiTCgHT7z2IM7UgH4MA_s6XGU" />
         <param name="API_KEY_FOR_IOS" value="AIzaSyCri8p_Qh5jfbLj4ja0S1Z2Gx17PNdFR_o" />
     </gap:plugin>
+    <engine name="android" spec="^6.3.0" />
+    <engine name="ios" spec="^4.5.1" />
+    <plugin name="cordova-plugin-console" spec="^1.1.0" />
+    <plugin name="cordova-plugin-device" spec="^1.1.4" />
+    <plugin name="cordova-plugin-geolocation" spec="^2.4.3" />
+    <plugin name="cordova-plugin-googlemaps" spec="git+https://github.com/mapsplugin/cordova-plugin-googlemaps.git#multiple_maps">
```

Comment:
Remove lines 92-95. Using this plugin breaks the phone gap build. It was probably added back because of the command but since the plugin is now on your local machine, you don't need to keep it saved on the `config.xml` anymore.

> If you remove them and run the app again, it should still work. (do not delete the plugins folder)

Reply:
Removed